src/config/wmodules-office365.c

```diff
@@ -136,6 +137,35 @@ int wm_office365_read(__attribute__((unused)) const OS_XML *xml, xml_node **node
                     }
                     os_free(office365_auth->client_secret);
                     os_strdup(children[j]->content, office365_auth->client_secret);
+                } else if (!strcmp(children[j]->element, XML_API_TYPE)) {
+                    if (strlen(children[j]->content) == 0) {
+                        merror("Empty content for tag '%s' at module '%s'.", XML_API_TYPE, WM_OFFICE365_CONTEXT.name);
+                        OS_ClearNode(children);
+                        return OS_INVALID;
+                    }
+                    else if (!strcmp(children[j]->content, "commercial")) {
+                        os_free(office365_auth->login_fqdn);
+                        os_strdup("login.microsoftonline.com", office365_auth->login_fqdn);
+                        os_free(office365_auth->management_fqdn);
+                        os_strdup("manage.office.com", office365_auth->management_fqdn);
+                    }
+                    else if (!strcmp(children[j]->content, "gcc")) {
+                        os_free(office365_auth->login_fqdn);
+                        os_strdup("login.microsoftonline.com", office365_auth->login_fqdn);
+                        os_free(office365_auth->management_fqdn);
+                        os_strdup("manage-gcc.office.com", office365_auth->management_fqdn);
```

Comment:
Avoid using string literals. Create missing defines in `src/wazuh_modules/wm_office365.h` and use them.

Reply:
Resolved with #[0b77d45](https://github.com/wazuh/wazuh/pull/15010/commits/0b77d4531d5498a82f512ba221b22891a8b61a95), defines added with [b43b448](https://github.com/wazuh/wazuh/pull/15010/commits/b43b4485e3ef31576ba90257b98d9a4641c5e90c)